trade-finance-manager-api/src/graphql/reducers/deal.api-test.js

```diff
@@ -33,19 +38,19 @@ describe('reducer - deal', () => {
       },
     };
 
-    const result = dealReducer(mockDeal);
+    const result = await dealReducer(mockDeal);
 
     const expected = {
       _id: mockDeal._id,
-      dealSnapshot: mapDealSnapshot(mockDeal),
+      dealSnapshot: await mapDealSnapshot(mockDeal),
       tfm: mapDealTfm(mockDeal),
     };
 
     expect(result).toEqual(expected);
   });
 
   describe('when dealType is `GEF`', () => {
-    it('should return mapGefDeal', () => {
+    it('should return mapGefDeal', async () => {
```

Comment:
I am slightly concerned with high number of changes, can we for the purpose of this PR only update where necessary and then revert back to old code. We can then cover this under tech-debt.

What are your thoughts?

Reply:
discussed